orientation_covariance: [ 0.020354665, 0, 0, 
                          0, 5.72348e-05, 0, 
                          0, 0, 0.000119784 ]

linear_acceleration_covariance:  [ 0.000385, 0, 0, 
                                   0, 0.000385, 0, 
                                   0, 0, 0.000385 ]

angular_velocity_covariance: [ 0.00002742, 0, 0, 
                               0, 0.00002742, 0, 
                               0, 0, 0.00002742 ]

magnetic_field_covariance : [9.0e-8, 0, 0,
                             0, 9.0e-8, 0,
                             0, 0, 9.0e-8]